### DISCUSSION TITLE
Getting an error message that I expect is my fault rather than a bug.

### DISCUSSION THREAD

**Discussion by PigeonFX** [JarrettR/Stretch#29] [General]
When I hit the out button it throws this error 
![Capture](https://user-images.githubusercontent.com/39096995/210921926-d5847b37-69ab-4ecf-ab9b-9dfe72abcddc.PNG)

The only file that is generated in the folder is the log, and just contains:

```
Begin Stretch debug
Import BS4
E:\KiCad\3rdparty\plugins\com_github_jarrettr_stretch\kiplug
kicad_pcb

```
Using:
KiCad Version: (6.0.6), release build
Inkscape 1.2
Windows 10

**Comment by JarrettR** [comment]
Not your fault at all - See issue #27 
That bug is fixed in the last master version of this repo. I need to push the changes into new release, I've just been short on time.

If you download this repo as a zip and overwrite the files at `E:\KiCad\3rdparty\plugins\com_github_jarrettr_stretch\`, then I expect the issue to go away.

**Reply by PigeonFX** [reply]
Thank you, that does indeed fix the issue. 

Although that also brings up another question: how to edit the SVG. 

I'm coming from SVG-To-Shenzen where you just draw onto the corresponding layer, but Stretch has all the layers listed but everything is under a Modules folder, and say for example adding a shape to the F.Cu folder saving, bringing back into Kicad, closing and reopening results in no change to the F.Cu layer. Am I doing something wrong, or can you only edit within the Modules rather than the layers?

**Reply by JarrettR** [reply]
Drawing can't really be done from scratch in Inkscape, there is a lot of hidden additional metadata with each object that tells Stretch how to bring it back to KiCad.

Think of it more like object creation in KiCad, then bringing it into Inkscape for modification, and then back to KiCad.

**Reply by PigeonFX** [reply]
OK, will have to get my head around that workflow, I'm so used to drawing each board layer in Inkscape from scratch and then bringing it into KiCad with SVG-To-Shenzen and adding to it within KiCad and exporting Gerbers. 

Love the idea of being able to go back and forth between the two, so hopefully, I can get my head around a workflow for my use case which is recreating vintage hand-drawn PCBs for restoration and retrofits. 

I very much like the idea of starting with an actual schismatic and board file and taking that into Inkscape, as opposed to currently having to start with tracing in Inkscape and forcing it into KiCad in a oneway process.
